playbooks/setup_forklift.yml

```diff
@@ -31,6 +31,16 @@
           - rubygem-deep_merge
         state: 'present'
 
+    - name: 'install telemetry system dependencies'
```

Comment:
I'm not sure this is the right solution. Right now it's compiling the whole SDK which takes a few minutes. In CI we use wheels, so that's probably better.